src/main/java/com/atlassian/bitbucket/jenkins/internal/config/BitbucketPluginConfiguration.java

```diff
@@ -136,7 +139,28 @@ private void updateJobs(Map<String, String> oldBaseUrls) {
                                 }
                             }
                         });
+                Jenkins.get().getAllItems(MultiBranchProject.class)
+                        .forEach(project -> processMultibranchProject(project, changedServerIds));
             }
         }
     }
+
+    @SuppressWarnings("unchecked")
+    private void processMultibranchProject(MultiBranchProject project, Set<String> changedServerIds) {
```

Comment:
/nit - `processMultiBranchProject` to match `MultiBranchProject` casing

Reply:
I will refactor all instances to be "Multibranch" rather than "MultiBranch" to be consistent with the rest of Jenkins